metadata/boundaries/county_boundaries

```diff
@@ -0,0 +1,82 @@
+# Title
+
+Utah County Boundaries
+
+## ID
+
+b2c8af56-3d92-4aa7-ac2f-b1388051d9ce
+
+## Brief Summary
+
+A statewide polygon dataset of county boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+This dataset contains polygons for all twenty-nine counties in Utah, where the perimeter of the polygon represents the county boundary. Each polygon also contains county population as of the [2020 Census](https://www.census.gov/library/stories/state-by-state/utah-population-change-between-census-decade.html) and current approximate population.
+
+## Description
+
+### What is the dataset
+
+This dataset represents each county as a polygon with information such as the Entity Number and FIPS code. A full explanation of the field definitions for this dataset is available on the [schema summary](https://docs.google.com/document/d/1e98nKP6PW9tuTy3nZYS4qhMp9fC8E6uv73SVbBOYvsk/edit?usp=sharing) document. The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 meter coordinate precision).
```

Comment:
I know the spatial reference is in a lot of the datasets metadata

> The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 meter coordinate precision).

This is the same for every dataset stored in the SGID. Is it a relic or does it still provide value? Should it be promoted to it's own field rather than sometimes nested in description text?

Reply:
Good question. I suppose it depends on how much time the average user spends on our site. For example, if the average user only goes to our website for a single dataset, downloads it, then leaves, then I think it's worth mentioning on every data page since that user won't likely realize that this sentence is on every layer in the SGID. This is the treat-every-dataset-like-it's-the-only-one-they'll-see approach. On the flipside, if the average user is quite familiar with our data and makes many visits to the site, then perhaps it is a relic and isn't helpful because the user will already know that all of our data are in NAD 1983 UTM Zone 12N. 

If we do end up adding the coordinate system to every data page in the SGID, I do think it would be helpful to include it in its own heading, like how the License is almost always the same on every dataset.

Reply:
Have you mentioned it on the other documents?

Reply:
Yes, but so far I have only been including it if it was part of the original metadata, which has been the case for some but not all of the layers I've worked on.